courses/templates/partials/faculty-carousel.html

```diff
@@ -0,0 +1,23 @@
+{% load wagtailimages_tags %}
+<div class="faculty-block">
+  <div class="container">
+    <div class="head">
+      <h1 class="text-uppercase">{{ course.faculty.heading }}</h1>
+      <h3>{{ course.faculty.subhead }}</h3>
+    </div>
+    <div class="faculty-slider">
+      {% for member in course.faculty.members %}
+        <div class="slide">
+          <div class="slide-holder">
+            {% image member.value.image fill-300x300 as faculty_image %}
```

Comment:
just a question. why these dimensions 300*300  ? did you check with some other dimensions too ?

Reply:
That's roughly five pixels over the size of image element in the carousel card. Having wagtail serve the image in fill mode gives a better chance to cover the area with the head shot image close to the center.